src/main/java/org/veupathdb/service/eda/ds/metadata/AppsMetadata.java

```diff
@@ -97,6 +102,8 @@ public class AppsMetadata {
           viz("heatmap", new HeatmapPlugin()),
           viz("map-markers", new MapPlugin()),
           viz("map-markers-overlay", new MapMarkersOverlayPlugin())),
+          viz("map-markers-bubbles", new BubbleMapMarkersPlugin()),
+          viz("map-markers-bubbles-legend", new BubbleMapMarkersLegendPlugin()),
```

Comment:
I don't think these two belong here or for the MBio.Maps section below.

One day maybe (as per Danielle's comment below), but for now I think it will add confusion.